tests/installation/ipxe_install.pm

```diff
@@ -234,12 +234,7 @@ sub run {
         unless (get_var('HOST_INSTALL_AUTOYAST')) {
             select_console 'installation';
             save_screenshot;
-            if (check_var('VIDEOMODE', 'ssh-x') or is_tumbleweed) {
-                enter_cmd_slow("yast.ssh");
-            }
-            elsif (check_var('VIDEOMODE', 'text')) {
-                enter_cmd_slow('DISPLAY= yast.ssh');
```

Comment:
You only need to modify this line ```enter_cmd_slow('DISPLAY= yast.ssh');```.  Basically, you do not need to remove this whole block. You can add a bug reference here.

Reply:
Is DISPLAY going to be added back? what does it  help at here? It was removed from TW by me in the past because it brought troubles too.

Reply:
I think you'd better keep the if eslif block which differentiate text mode from default non-text mode. There are product bugs for this issue.

Reply:
Yes, it should be a bug.

I reviewed the ipxe bootscript again, found that actually DISPLAY is useless for textmode installation.  The bootscript was configured by
```
$cmdline_extra .= " ssh=1 sshpassword=$testapi::password ";
$cmdline_extra .= " vnc=1 VNCPassword=$testapi::password " unless check_var('VIDEOMODE', 'text');
```
So for textmode installation, there is no VNC or X server. DISPLAY='' is not needed.

Reply:
would you please add a comment with bug id to explain the background of removing it?

Reply:
1. Boot commands on current linux command line and ipxe script are different. The ways in which YasT opened are different. So it is different about how to turn on/off text mode
2. You should not remove the whole block. You just need to modify that single line and add bug info.

Reply:
> 1. Boot commands on current linux command line and ipxe script are different. The ways in which YasT opened are different. So it is different about how to turn on/off text mode

What do you mean about "Boot commands on current linux command line" and "Boot command of ipxe script"? what's the different between them?

Reply:
> would you please add a comment with bug id to explain the background of removing it?

No problem

Reply:
> > 1. Boot commands on current linux command line and ipxe script are different. The ways in which YasT opened are different. So it is different about how to turn on/off text mode
> 
> What do you mean about "Boot commands on current linux command line" and "Boot command of ipxe script"? what's the different between them?

Yast open directly with IPXE+IPMI. But need to ssh and type DISPLAY= yast.ssh the other way. So they are different.

Reply:
so that means yast.ssh has it own bootscript which might include the original bootscript and add some special logic? then I understand.